src/libraries/System.IO.Compression/src/System/IO/Compression/DeflateZLib/ZLibNative.cs

```diff
@@ -64,11 +64,18 @@ public enum ErrorCode : int
         ///    <code>int memLevel = 7;</code> <br />
         ///    <code>ZLibNative.CompressionStrategy strategy = ZLibNative.CompressionStrategy.DefaultStrategy;</code> </p>
         /// </summary>
+        ///
+        /// <p><em>Smallest Size Compression:</em></p>
+        /// <p><code>ZLibNative.CompressionLevel compressionLevel = ZLibNative.CompressionLevel.BestCompression;</code> <br />
+        ///    <code>int windowBits = 15;  // or -15 if no headers required</code> <br />
+        ///    <code>int memLevel = 8;</code> <br />
+        ///    <code>ZLibNative.CompressionStrategy strategy = ZLibNative.CompressionStrategy.DefaultStrategy;</code> </p>
```

Comment:
nit: `</summary>` goes after `</p>`:
```suggestion
        ///
        /// <p><em>Smallest Size Compression:</em></p>
        /// <p><code>ZLibNative.CompressionLevel compressionLevel = ZLibNative.CompressionLevel.BestCompression;</code> <br />
        ///    <code>int windowBits = 15;  // or -15 if no headers required</code> <br />
        ///    <code>int memLevel = 8;</code> <br />
        ///    <code>ZLibNative.CompressionStrategy strategy = ZLibNative.CompressionStrategy.DefaultStrategy;</code> </p>
        /// </summary>
```